### PR TITLE
react-native Dimensions: Added addEventListener/removeEventListener

### PR DESCRIPTION
**Pull request by adrianhall**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X\] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: https://facebook.github.io/react-native/docs/dimensions.html
- [X] Increase the version number in the header if appropriate.

### CODE REVIEW

File: types/react-native/index.d.ts [hunk 1]
```diff
@@ -1,11 +1,12 @@
-// Type definitions for react-native 0.47
+// Type definitions for react-native 0.48
```
Comment (iRoachie) [inline types/react-native/index.d.ts]: Remove this. 0.48 ins't out as yet.

File: types/react-native/index.d.ts [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
 // Project: https://github.com/facebook/react-native
 // Definitions by: Eloy Durán <https://github.com/alloy>
 //                 Fedor Nezhivoi <https://github.com/gyzerok>
 //                 HuHuanming <https://github.com/huhuanming>
 //                 Kyle Roach <https://github.com/iRoachie>
 //                 Tim Wang <https://github.com/timwangdev>
 //                 Kamal Mahyuddin <https://github.com/kamal>
+//		   Adrian Hall <https://github.com/adrianhall>
```
Comment (iRoachie) [inline types/react-native/index.d.ts]: Adding your name here means that you become a maintainer and all issues and pull requests will be sent to your email as well. Just making sure you know what this means.

**Review by iRoachie — CHANGES_REQUESTED** [review]

File: types/react-native/index.d.ts (hunk 1 above)
Comment (adrianhall) [inline types/react-native/index.d.ts]: Will do

File: types/react-native/index.d.ts (hunk 2 above)
Comment (adrianhall) [inline types/react-native/index.d.ts]: Did not realize that - should probably back that out here.  Next commit will have the right thing here.

**Comment by iRoachie** [comment]
👍🏽

**Pull request merged**